function throwError() {
    throw new Error('Error thrown from exceptionScenarios.js!');
}